src/pages/guides/webpack-configuration.md

```diff
@@ -0,0 +1,194 @@
+# Webpack Configuration 
+
+## Overview
+
+App Builder bundles Runtime action code using Webpack. Users can specify a Webpack configuration for an individual action, a set of actions, 
+or an entire project. See [here](https://webpack.js.org/configuration/#options) for Webpack configuration options. 
+
+## Configuration File 
+
+When Runtime code is bundled, the CLI will search for a `*webpack-config.js` file starting at the path of the action code, working up to the root 
```

Comment:
might want to make this more specific to the app-plugin the CLI doesn't really do much ...
When action code is bundled during `aio app build|run|deploy` the action directory is searched for a webpack config file named `*webpack-config.js`. The search is done first in the directory of the action being built, then the parent directory, up to the project root.